docs/howto/custom-jupyterhub-image.md

```diff
@@ -39,8 +39,23 @@ flowchart TD
   -- More testing for hubs that are configured differently ---
   deploy_everywhere --> end_condition[fa:fa-exclamation `hub-experimental` image is the same as `hub`]
 ```
+## How to create a new hub experimental image
 
-## How to install an unreleased version of a package in the `hub-experimental` image
+1. You will first need to create a new `.txt` file with a name relevant for the experiment, let's say `new-experiment-requirements.txt`.
```

Comment:
Should we also add an item here about creating the repository on quay.io?

Reply:
I noticed that the quay repos actually get created when running `chartpress`. But they get created private and you still need to go to the website to change its privacy because the org is not allowed to have any private repos. 

I think it makes sense to add the step to create a public repo on quay before building and pushing. Thank you!!!